kalite/contentload/management/commands/generate_assessment_zips.py

```diff
@@ -56,7 +60,9 @@ def download_urls(zf, urls):
 
     urls = set(urls)
 
-    pool = ThreadPool(5)
+    # TODO(jamalex): this can be changed to 5 or so during download, but multiple threads writing to the zip
+    # at the same time leads to corruption, so you'll then need to re-run with it set to 1
+    pool = ThreadPool(1)
     download_to_zip_func = lambda url: download_url_to_zip(zf, url)
```

Comment:
Interesting, I guess we can split the downloading part to be concurrent and let the writing-to-zip part just happen on the main thread. Can you make such a refactoring a todo here?